File: Python/Program/Shirt.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

## Import file
from Clothing import Clothing

## Inheritance dengan file Clothing
class Shirt(Clothing):
    ## Atribut privat
     __colour = ""
     __sleeveType = ""
     
     ## Constructor
    ## Atribut Publik
     def __init__(self, colour = "", sleeveType = ""):
          self.__colour = colour
          self.__sleeveType = sleeveType
          
    ## Getter dan Setter    
          
     def get_colour(self):
         return self.__colour
     def set_colour(self, colour):
         self.__colour = colour
         
     def get_sleeve(self):
         return self.__sleeveType
     def set_sleeve(self, sleeveType):
         self.__sleeveType = sleeveType      
          
    
        
        
    

File: CPP/Program/Shirt.cpp
/*Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
Desain Pemrograman Berbasis Objek untuk keberkahanNya maka saya tidak 
melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.*/


//import library
#include <iostream>
#include <string>
#include "Clothing.cpp" //import file clothing

//menggunakan namespace standar
using namespace std;

// Inheritance dengan file clothing
class Shirt : public Clothing{
    // Atribut privat
    private:
        string colour;
        string sleeve_type;
    
    // Atribut Publik
    public:
    // Constructor
        Shirt(){
            this->colour = "";
            this->sleeve_type = "";
        }
    // Constructor dengan parameter
        Shirt(string colour, string sleeve_type){
            this->colour = colour;
            this->sleeve_type = sleeve_type;
        }

    // Getter dan Setter
        string getColour(){
            return this->colour;
        }

        void setColour(string colour){
            this->colour = colour;
        }

        string getSleeve(){
            return this->sleeve_type;
        }

        void setSleeve(string sleeve_type){
            this->sleeve_type = sleeve_type;
        }

        ~Shirt(){
            
        }
    
};
